app-multimedia/feishin/autobuild/overrides/usr/share/applications/feishin.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Feishin
+Comment=A music player client for self-hosted streaming services
+Comment[zh_CN]=自建音乐串流服务客户端
```

Comment:
Feishin fails to detect mpv that is already installed on the system. Yes, the upstream already noted this, but it seems silly for a user to do things that the distro should have already handled (setting the mpv path before the first launch).

Upstream statement:

"Upon startup you will be greeted with a prompt to select the path to your MPV binary. If you do not have MPV installed, you can download it [here](https://mpv.io/installation/) or install it using any package manager supported by your OS. After inputting the path, restart the app."

What the user sees:

```
MPV REQUIRED

Set your MPV executable location below and restart the application.
```

Yes, the user could have went ahead and found the upstream repo, and read their README. But why would we assume for anyone to have thought of doing so? We packaged Feishin using our *system dependency*, mpv, and it should be detected right away - otherwise, why should the dependency be there in the first place?

As it stands, I suggest leaving this package in this topic (I have removed the previous packages from `stable`); another way to handle this would be to patch in a `localSetting` item, `mpv_path`, per [/src/renderer/features/action-required/components/mpv-required.tsx:10](https://github.com/jeffvli/feishin/blob/1acfa93f1addb004de3734c2c1486279c2bec716/src/renderer/features/action-required/components/mpv-required.tsx#L10C9-L10C24).